CHANGELOG.md

```diff
@@ -8,30 +8,52 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+-
+
+### Fixed
+-
+
+### Changed
+-
+
+### Removed
+-
+
+### Deprecated
+-
+
+## [0.3.0]
+
+### Added
+
 - Added Kernelised Stein Discrepancy divergence in `coreax.metrics.KSD`.
 - Added the `coreax.solvers.recombination` module, which provides the following new solvers:
   - `RecombinationSolver`: an abstract base class for recombination solvers.
   - `CaratheodoryRecombination`: a simple deterministic approach to solving recombination problems.
   - `TreeRecombination`: an advanced deterministic approach that utilises `CaratheodoryRecombination`,
     but is faster for solving all but the smallest recombination problems.
-- Added supervised coreset construction algorithm in `coreax.solvers.GreedyKernelPoints`
-- Added `coreax.kernels.PowerKernel` to replace repeated calls of `coreax.kernels.ProductKernel`
-within the `**` magic method of `coreax.kernel.ScalarValuedKernel`
-- Added scalar-valued kernel functions `coreax.kernels.PoissonKernel` and `coreax.kernels.MaternKernel`
+- Added supervised coreset construction algorithm in `coreax.solvers.GreedyKernelPoints`.
+- Added `coreax.kernels.PowerKernel` to replace repeated calls of `coreax.kernels.ProductKernel`.
+within the `**` magic method of `coreax.kernel.ScalarValuedKernel`.
+- Added scalar-valued kernel functions `coreax.kernels.PoissonKernel` and `coreax.kernels.MaternKernel`.
 - Added `progress_bar` attribute to `coreax.score_matching.SlicedScoreMatching` to enable or
 disable tqdm progress bar terminal output. Defaults to disabled (`False`).
 - Added analytical tests for kernel herding, and associated documentation in `tests.unit.test_solvers`.
-
+- Added a custom version of `jnp.atleast_2d` that converts one dimensional n-vectors to shape (n,1).
```

Comment:
If this refers to `_atleast_2d_consistent`, it shouldn't be in the changelog, as it's a private function.